test/wasmtime_vm_test.cc

```diff
@@ -0,0 +1,230 @@
+// Copyright 2016-2019 Envoy Project Authors
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <string>
+
+#include "include/proxy-wasm/context.h"
+#include "include/proxy-wasm/wasm.h"
+#include "include/proxy-wasm/wasmtime.h"
+
+#include "wasmtime/crates/c-api/include/wasmtime.h"
+#include "wasmtime/include/wasm.h"
+
+#include "test/watdata.h"
+#include "gtest/gtest.h"
+
+namespace proxy_wasm {
+namespace {
+
+TEST(Wasmtime, CreateVM) {
+  auto vm = proxy_wasm::createWasmtimeVm();
+  EXPECT_EQ(vm->cloneable(), proxy_wasm::Cloneable::CompiledBytecode);
+  EXPECT_EQ(vm->runtime(), "wasmtime");
+}
```

Comment:
Those tests should be executed across all runtimes, where possible, to guarantee consistent behavior.
See: https://github.com/envoyproxy/envoy/blob/436e7cd34d5eb3d7fc95673bea458d630f5cbd60/test/extensions/common/wasm/wasm_test.cc#L94-L102

Reply:
resolved in https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/73/commits/abae5f08df6731536a0f7d820bba8c1b3ed0c1d0